src/transformers/models/herbert/tokenization_herbert_fast.py

```diff
@@ -22,10 +22,7 @@
 
 logger = logging.get_logger(__name__)
 
-VOCAB_FILES_NAMES = {
-    "vocab_file": "vocab.json",
-    "merges_file": "merges.txt",
-}
+VOCAB_FILES_NAMES = {"vocab_file": "vocab.json", "merges_file": "merges.txt", "tokenizer_file": "tokenizer.json"}
```

Comment:
It's not really related to the current PR but I "needed" to add it for my tests to pass. 

Most other tokenizers have this argument listed in `VOCAB_FILES_NAMES`, and it seems to me that it is necessary for the `from_pretrained` method to take into account the `tokenizer_file` file located in the folder. If you share this opinion, we should probably take the opportunity to regulate the other fast tokenizers such as `ConvBERT` for which this argument is missing.

Reply:
Nice fix